CONTRIBUTING.md

```diff
@@ -43,28 +43,28 @@ you can follow the steps below to test your changes:
 
 1. Create the cluster:
 
-```sh
```

Comment:
@m1kola, this is merged in another PR, right?
Could we rebase it with the main and have only the second commit?